kv/transaction/mvcc/transaction_test.go

```diff
@@ -106,6 +106,22 @@ func TestGetLock4A(t *testing.T) {
 	assert.Equal(t, lock, *gotLock)
 }
 
+func TestGetEmptyLock4A(t *testing.T) {
+	lock := Lock{
+		Primary: []byte{16},
+		Ts:      100,
+		Ttl:     100000,
+		Kind:    WriteKindRollback,
+	}
+	txn := testTxn(42, func(m *storage.MemStorage) {
+		m.Set(engine_util.CfLock, []byte{1}, lock.ToBytes())
+	})
+
+	gotLock, err := txn.GetLock([]byte{2})
```

Comment:
let's merge it to `TestGetLock4A`

Reply:
Ok, I'm already merge it to `TestGetLock4A `.